charts/wga/values.yaml

```
@@ -7,6 +7,7 @@
 ## @param global.imageRegistry Global Docker image registry
 global:
   imageRegistry: "ghcr.io/kraudcloud"
+  privateImageRegistry: "ctr.0x.pt"
```

Just noticed this registry is missing the repository name, which results in this error
`Failed to pull image "ctr.0x.pt/wga-welcome:main": failed to pull and unpack image "ctr.0x.pt/wga-welcome:main": failed to resolve reference "ctr.0x.pt/wga-welcome:main": unexpected status from HEAD request to https://ctr.0x.pt/v2/wga-welcome/manifests/main: 400 Bad Request`.

---

ah it should be:
```suggestion
  privateImageRegistry: "ctr.0x.pt/wga"
```
?